TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/ScoringAlgorithm.java

```diff
@@ -44,7 +48,9 @@ public void init() {
 
     }
 
-    private void scoring() {
+
+
+    public void scoring() {
```

Comment:
this isn't functional as a proof of concept if you can't call this method anywhere